test/screenshot/spec/mdc-drawer/classes/permanant-below-top-app-bar.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
```

Comment:
We might not need to screenshot test this. Dismissible & permanent drawers have exact same styles except the JS logic.

Reply:
true - i'll remove